src/main/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoal.java

```diff
@@ -48,8 +48,8 @@ public static int shouldHaveRead(int booksToReadThisYear) {
      * @return the number of books that should have been read a week (on average) from the start of
      * the year
      */
-    public static int booksToReadFromStartOfYear(int booksToReadThisYear) {
-        return ((int) Math.ceil(booksToReadThisYear / WEEKS_IN_YEAR));
+    public static double booksToReadFromStartOfYear(int booksToReadThisYear) {
+        return (double)((float)booksToReadThisYear / WEEKS_IN_YEAR);
```

Comment:
Could you please remove the first `double` cast and then replace the `float` cast with a `double`. We should get:
```java
        return ((double) booksToReadThisYear / WEEKS_IN_YEAR);
```

Reply:
Done.